package.json

```diff
@@ -1,35 +1,33 @@
 {
   "name": "fork-sync",
-  "version": "2.0.10",
   "private": true,
   "description": "Keep your fork up to date",
   "main": "lib/main.js",
+  "license": "UNLICENSED",
   "scripts": {
+    "prepare": "husky install",
     "build": "tsc",
```

Comment:
note: we need to bundle now that we removed node_modules. In our other actions we use [@vercel/ncc](https://www.npmjs.com/package/@vercel/ncc)

Reply:
example of one of our actions that does this: https://github.com/ExodusMovement/request-permission-action/blob/667993ba6aea52210dbcab9eade2fbbd049fab67/package.json#L18

Reply:
I updated the implementation